k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -269,6 +269,14 @@ public boolean allHeadersVisible() {
         return mMessageView.getMessageHeaderView().additionalHeadersVisible();
     }
 
+    public boolean isAccountReportSpamEnabled() {
+        try {
+            return (mAccount != null && !mAccount.getReportSpamRecipient().isEmpty());
+        } catch (Exception e) {
+            return false;
```

Comment:
What exception is likely here? This feels like it should never throw. And should it not be logged?